sdk/servicebus/azure-servicebus/azure/servicebus/_common/message.py

```diff
@@ -860,12 +882,148 @@ def locked_until_utc(self):
+    :paramtype data_body: Union[str, bytes, List[Union[str, bytes]]]
+    :keyword sequence_body: The body consists of one or more sequence sections and
+     each section contains an arbitrary number of structured data elements.
+    :paramtype sequence_body: Union[List[Any], List[List[Any]]]]
+    :keyword value_body: The body consists of one amqp-value section and the section contains a single AMQP value.
+    :paramtype value_body: Any
+    :keyword header: The amqp message header. This must be a dictionary with the following
+     keys:
+
+        - `delivery_count` (int)
+        - `time_to_live` (int)
+        - `durable` (bool)
+        - `first_acquirer` (bool)
+        - `priority` (int)
+
+    :paramtype header: dict
+    :keyword footer: The amqp message footer.
+    :paramtype footer: dict
+    :keyword properties: Properties to add to the amqp message. This must be a dictionary with the following
+     keys:
+
+        - message_id (str or bytes),
+        - user_id (str or bytes),
+        - to (Any),
+        - subject (str or bytes),
+        - reply_to (Any),
+        - correlation_id (str or bytes),
+        - content_type (str or bytes),
+        - content_encoding (str or bytes),
+        - creation_time (int),
+        - absolute_expiry_time (int),
+        - group_id (str or bytes),
+        - group_sequence (int),
+        - reply_to_group_id (str or bytes)
+
+    :paramtype properties: dict
+    :keyword application_properties: Service specific application properties.
+    :paramtype application_properties: dict
+    :keyword annotations: Service specific message annotations.
+    :paramtype annotations: dict
+    :keyword delivery_annotations: Service specific delivery annotations.
+    :paramtype delivery_annotations: dict
     """
 
-    def __init__(self, message):
-        # type: (uamqp.Message) -> None
-        self._message = message
+    def __init__(self, **kwargs):
+        # type: (Any) -> None
+        self._message = kwargs.pop("message", None)
+
+        if self._message:
+            # internal usage only for service bus received message
+            return
+
+        self._data_body = kwargs.pop("data_body", None)
+        self._sequence_body = kwargs.pop("sequence_body", None)
+        self._value_body = kwargs.pop("value_body", None)
+
+        validation = [
+            body
+            for body in (self._value_body, self._data_body, self._sequence_body)
+            if body is not None
+        ]
+        if len(validation) != 1:
+            raise ValueError(
+                "There should be one and only one of either data_body, sequence_body "
+                "or value_body being set as the body of the AMQPAnnotatedMessage."
+            )
+
+        header = kwargs.get("header")
+        footer = kwargs.get("footer")
+        properties = kwargs.get("properties")
+        application_properties = kwargs.pop("application_properties")
```

Comment:
was this meant to be a `get`? otherwise, I believe this will result in an error if `application_properties` isn't included

Reply:
thanks! fixed